main.js

```diff
@@ -0,0 +1,12 @@
+const timer = document.getElementById('timer');
+
+window.onload = () => {
+    let timerCount = localStorage.getItem('timerCount') || 0;
+    timer.textContent = timerCount;
+       
+    setInterval(function(){
+        timerCount++;
+        timer.textContent = timerCount;
```

Comment:
You duplicated this row. Create functions for render counter and for increment counter